Gemfile

```diff
@@ -1,11 +1,15 @@
 source 'https://rubygems.org'
 
 # sufia stuff!
-gem 'sufia', '7.3.0'
+gem 'sufia', '7.4.0'
 gem 'blacklight_range_limit', ">= 6.1.2"
 gem 'riiif', '~> 1.4'
 # It may seem like this is a dependency upstream, but we need it here.
 # https://project-hydra.slackarchive.io/dev/page-100/ts-1491587437169731
+# Update: slackarchive permalink doesn't work after some max. search for 'rsolr' and look
+# at April 7 results for hints of the conversation? Unfortunately you can't get context.
+# I tried relaxing this with the sufia 7.4 upgrade but it a bug where submitting a record
+# edit form did not correctly load the redirect back to the show page.
 gem 'rsolr', '~> 1.0'
 gem 'active-fedora', '~>11.1.6'
```

Comment:
sufia will allow rsolr 2.0, but we found it actually only worked with rsolr 1.x, is that right?

Reply:
I don't really know. I had pasted a slackarchive link here, but it no longer works and as I wrote in the comment I can't get to the context around the discussion.

I do know that I tried allowing rsolr > 1.0 and observed the bug described, which went away when I replaced the rsolr line.

I didn't investigate farther, figured we didn't need to upgrade rsolr to upgrade rails, which was the primary goal of this work.

Reply:
Okay, thanks. It's distressing becuase there's another commit in Sufia where @mjgiarlo is specifically making it work _with_ rsolr 2.0.  Maybe it's something specific to our app rather than sufia "7.4" that makes it not. Hard to say. 

I think really what we're learning is that sufia is not only not maintained, but essentially not maintainable, things that seem like they should be simple won't be, we probably should never try to update sufia again, just live with it as it is, or painfully upgrade to hyrax (or abandon it). I ran into a similar thing trying to do a [simple dependency change on curation_concerns](https://github.com/samvera/curation_concerns/pull/1186), and then basically felt like giving up.